app/cdap/components/PostRunActions/WizardNav/WizardConfigureConfirmStep/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button, Tooltip, withStyles } from '@material-ui/core';
+import { MyArtifactApi } from 'api/artifact';
+import { MyPipelineApi } from 'api/pipeline';
+import ConfigurationGroup from 'components/shared/ConfigurationGroup';
+import React, { useEffect, useState } from 'react';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import styled from 'styled-components';
+import uuidV4 from 'uuid/v4';
+
+export interface IWizardConfigureConfirmStepProps {
+  state: any;
```

Comment:
Try not to use just `state` its confusing and one of the keywords of react - wizardPanelState?